packages/lesswrong/components/localGroups/GroupFormDialog.tsx

```diff
@@ -113,6 +115,7 @@ const GroupFormDialog =  ({ onClose, classes, documentId }: {
         formComponents={{
           FormSubmit: SubmitComponent
         }}
+        removeFields={isEAForum ? ['types'] : []}
```

Comment:
...Huh, I didn't know `removeFields` existed. I would still probably recommend specifying `hidden: isEAForum` in the schema for types.